packages/dev/core/src/Shaders/ShadersInclude/pbrDebug.fx

```diff
@@ -128,6 +128,14 @@ if (vClipSpacePosition.x / vClipSpacePosition.w >= vDebugMode.x) {
         gl_FragColor.rgb = subSurfaceOut.transmittance;
     #elif DEBUGMODE == 70 && defined(SUBSURFACE) && defined(SS_REFRACTION)
         gl_FragColor.rgb = subSurfaceOut.refractionTransmittance;
+    #elif DEBUGMODE == 72
+        gl_FragColor.rgb = vec3(1.0-roughness); // TODO
+    #elif DEBUGMODE == 73
+        gl_FragColor.rgb = vAlbedoColor.rgb; // TODO
+    #elif DEBUGMODE == 74
+        gl_FragColor.rgb = vReflectivityColor.rgb; // TODO

```

Comment:
This is the specular color of the material in a specular workflow only. same as before it looks like it is what you want but just to be sure ?

In a metallic workflow, rg are the metallic roughness chanels https://github.com/BabylonJS/Babylon.js/blob/841bdc70820fe8c26f369e60964c7dac8543669d/packages/dev/core/src/Shaders/ShadersInclude/pbrBlockReflectivity.fx

If the debug is for specular raw value only it should not be possible to pick if METALLICWORKFLOW has been defined.

If the debug is for the overall specularity, after input management and products with textures, you could use: `gl_FragColor.rgb = surfaceReflectivityColor;` 

Also this would benefit from the `#define DEBUGMODE_GAMMA` cause the specular color is in linear space.